Android/3D Cave Runner - Android Studio/app/src/main/assets/custom_theme.json

```diff
@@ -1 +1,355 @@
+    "bracketPrizeViewZImage":"vc_tile.png",
+    "bracketPrizeViewCashImage":"cash_tile.png",
+    "bracketPrizeViewLiveEventImage":"vc_custom.png",
+    "backgroundImage":"cave_background.jpg"
+  },
+  "version":{
+    "SkillzOTAVersion":null,
+    "type":"Ignore",
+    "description":"Version of this OTA package, set at runtime via a file generated at compile time."
+  },
+  "currencyValues":{
+    "practiceCurrencyString":"🅩",
+    "practiceCurrencySimpleIconURI":"ic_z_black_med_SKZ@2x.png",
+    "practiceCurrencyFullIconLargeURI":"ic_z_white_large_SKZ@2x.png",
+    "description":"value"
+  },
+  "opacityLevels":{
+    "opacityPrimary":0.9,
+    "opacitySecondary":0.8,
+    "opacityTertiary":0.7,
+    "opacityNone":1.0,
+    "description":0.0
+  },
+  "enums":{
+    "sidePanelBlurStyle":1,
+    "statusBarContentStyle":0
+  },
+  "gradients":{
+    "prizeViewLiveEventGradient":[
+      "#000000",
+      "#000000"
+    ],
+    "prizeViewCashGradient":[
+      "#000000",
+      "#000000"
+    ],
+    "prizeViewZGradient":[
+      "#000000",
+      "#000000"
+    ],
+    "prizeViewTicketzGradient":[
+      "#f6604d",
+      "#e05745"
+    ],
+    "bracketPrizeViewLiveEventGradient":[
+      "#000000",
+      "#000000"
+    ],
+    "bracketPrizeViewCashGradient":[
+      "#000000",
+      "#000000"
+    ],
+    "bracketPrizeViewZGradient":[
+      "#000000",
+      "#000000"
+    ],
+    "btnGradientOne":[
+      "#732008",
+      "#ff4108"
+    ],
+    "btnGradientTwo":[
+      "#525151",
+      "#525151"
+    ],
+    "btnGradientThree":[
+      "#525151",
+      "#525151"
+    ],
+    "btnGradientStore":[
+      "#4fb031",
+      "#29782f"
+    ],
+    "playButtonGradient":[
+      "#732008",
+      "#ff4108"
+    ],
+    "textTicketzGradient":[
+      "#f87861",
+      "#e1654d"
+    ],
+    "textCashGradient":[
+      "#6ac141",
+      "#3da142"
+    ],
+    "textZGradient":[
+      "#4686ad",
+      "#2f6d98"
+    ],
+    "textNeutralGradient":[
+      "#b6b6b6",
+      "#6c6c6c"
+    ],
+    "overlayEntryContainerGradient":[
+      "rgba(39,39,39,.90)",
+      "rgba(39,39,39,.90)"
+    ],
+    "backgroundViewGradient":[
+      "#272727",
+      "#424242"
+    ],
+    "sideMenuHighlightedGradient":[
+      "#bd3100",
+      "#bd3100"
+    ],
+    "cashIconGradient":[
+      "#e9ab24",
+      "#ffe6c5"
+    ],
+    "practiceCurrencyIconGradient":[
+      "#5d7696",
+      "#6da5ce"
+    ],
+    "profileIconGradient":[
+      "#bd3100",
+      "#bd3100"
+    ],
+    "progressViewDarkGradient":[
+      "#272727",
+      "#484858"
+    ],
+    "navigationBarGradient":[
+      "#000000",
+      "#424242"
+    ],
+    "selectedCellBackgroundGradient":[
+      "#bd3100",
+      "#bd3100"
+    ],
+    "pointsPrizeGradient":[
+      "#bd3100",
+      "#bd3100"
+    ],
+    "tauntImageGradient":[
+      "#f48e30",
+      "#f48e30"
+    ],
+    "description":"gradient"
+  },
+  "lastUpdated":"2018-06-14T23:17:26Z",
+  "end":"skillz"
+}
```

Comment:
Are we no longer encrypting the custom theme file? We could still sign it, if desired.

Reply:
the theme file that was in before was invalid and caused a crash because it was missing a gradient value, I just pasted this one in so that it would run and I didn't feel the need to do the whole encryption thing again, i guess we could if we want to though